applications/crossbar/priv/couchdb/schemas/users.json

```diff
@@ -474,12 +474,20 @@
             "required": false,
             "type": "boolean"
         },
-        "vm_to_email_enabled": {
-            "default": true,
-            "description": "Determines if the user would like voicemails emailed to them",
-            "name": "VM To Emain Enabled",
+        "voicemail": {
```

Comment:
Why remove the `vm_to_email_enabled` flag? That is used by notify and teletype (via `kzd_user`) to determine whether to send the email notification. I'm not opposed to seeing it added as a sub-property of the "voicemail" key here, but it shouldn't be out-right removed.